tracpro/baseline/forms.py

```diff
@@ -54,32 +54,45 @@ def clean(self, *args, **kwargs):
         return cleaned_data
 
 
+class CustomModelChoiceField(forms.ModelChoiceField):
```

Comment:
What do you think about calling this something more specific like `QuestionModelChoiceField`?